cmd/new-ui/v1beta1/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
+	common_v1beta1 "github.com/kubeflow/katib/pkg/common/v1beta1"
+	ui "github.com/kubeflow/katib/pkg/new-ui/v1beta1"
+)
+
+var (
+	port, host, buildDir, dbManagerAddr *string
+)
+
+func init() {
+	port = flag.String("port", "80", "The port to listen to for incoming HTTP connections")
```

Comment:
Can we use another port for it? 80 is too common I think.

Reply:
@gaocegege @kimwnasptd Let's use 8080 here by default, since we are using 8080 in the deployment YAML: https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/ui/deployment.yaml#L29.

Reply:
ACK!

Reply:
pushed a commit that changes the port to 8080